config/settings.yml

```diff
@@ -54,7 +65,22 @@ exception_notifier:
 
 # Worker count
 workers:
-  hets: 4
+  hets: 1
+
+# The following paths can be absolute paths
+# or relative paths to the project root
+paths:
+  # General data.
+  data: data
+  # git repositories (names are numbers/ids)
+  git_repositories: data/repositories
+  # named symlinks to the git repositories
+  symlinks: data/git_daemon
+  # cache for files that needed to be checked out from the git repositories
+  commits: data/commits
+  # home directory of the git user
+  #   needed for handling of ssh keys in ~git/.ssh/authorized_keys
+  git_home: /home/git
```

Comment:
Actually we at least need the settings key `git_home`. We want to set it to `tmp/git` in the development and test environment. The default value is now `~git` and it is not expanded.